src/ResourceManagement/AppService/Microsoft.Azure.Management.AppService.Fluent/FunctionAppImpl.cs

```diff
@@ -157,5 +164,138 @@ public FunctionAppImpl WithLatestRuntimeVersion()
         {
             return WithRuntimeVersion("latest");
         }
+
+        public string GetMasterKey()
+        {
+            return GetMasterKeyAsync().ConfigureAwait(false).GetAwaiter().GetResult();
+        }
+
+        public async Task<string> GetMasterKeyAsync(CancellationToken cancellationToken = default(CancellationToken))
+        {
+            string apiVersion = "2016-08-01";
+            // Tracing
+            bool _shouldTrace = ServiceClientTracing.IsEnabled;
+            string _invocationId = null;
```

Comment:
Could you add comments why we have this generated snippet here and we do not call underlying generated client method?